lua/packer/plugin_types/local.lua

```diff
@@ -44,8 +50,16 @@ local function setup_local(plugin)
     end)
   end
 
-  plugin.updater = function(_)
+  plugin.updater = function(disp)
     return async(function()
+      disp:task_update(plugin_name, 'checking symlink...')
+      local resolved_path = vim.fn.resolve(to)
+      if resolved_path ~= from then
+        disp:task_update(plugin_name, 'updating symlink...')
+        local r = await(unlink(to)):and_then(symlink(from, to, { dir = true }))
+        return r
```

Comment:
Nitpick: we should either define `r` as `result.ok()` outside of this conditional or just straight return the `await()` call here.

Reply:
This can be cleaned up